src/main/java/cc/hyperium/Hyperium.java

```diff
@@ -111,6 +110,7 @@ public void init(InitializationEvent event) {
         EventBus.INSTANCE.register(CrosshairMod.getInstance());
         EventBus.INSTANCE.register(CONFIG.register(FPSLimiter.getInstance()));
         EventBus.INSTANCE.register(perspective = new PerspectiveModifierContainer());
+        EventBus.INSTANCE.register(new NameHistoryGui());
```

Comment:
Probably not good practice to leave an entire gui object in memory simply for a keybind.